common/testing/debugtimeout/debugtimeout.go

```diff
@@ -0,0 +1,18 @@
+//go:build test_dep || integration
```

Comment:
I don't know if we want to include `integration`. We don't use it in OSS. But I know consumers of the Server use it. So we might as well make it easy for them to use this as-is.